src/Loop/Driver.php

```diff
@@ -285,4 +311,39 @@
      * @return null|object|resource The loop handle the event loop operates on. `null` if there is none.
      */
     abstract public function getHandle();
+
+    /**
+     * Create a watcher identifier.
+     *
+     * Any watcher identifiers returned to clients by the driver MUST be created using this method. The returned watcher
```

Comment:
I think we can also remove the `returned to clients`, so we end up with `Every watcher identifier MUST be created using this method.`